src/modules/Elsa.Studio.Workflows.Core/Domain/Services/DefaultActivityRegistry.cs

```diff
@@ -53,7 +53,6 @@ public IEnumerable<ActivityDescriptor> List()
     {
         // Return the latest version of each activity descriptor from _activityDescriptors.
         return _activityDescriptors.Values
-            .Where(x => x.IsBrowsable)
```

Comment:
What may have been the reason for removing this filter?

Reply:
Within the call chain this filter is applied multiple times, so this felt like overkill.

Reply:
I see, good.